decepticonlp/extractor/basic.py

```diff
@@ -0,0 +1,36 @@
+import abc
+import random
+
+
+class ImportantWordExtractor(metaclass=abc.ABCMeta):
+    """
+        An abstract class used to represent the Keyword Extractor.SubClass should implement the extract method.
+        Methods
+        -------
+        extract(self, words: list, top_k = 1, **kwargs)
+            extracts top_k words and returns their index.
+    """
+
+    @abc.abstractmethod
+    def extract(self, words: list, **kwargs):
```

Comment:
`nit`
Missing argument docstrings?

Reply:
Also since this function should never be tested, you should add `pragma: no cover` in front of it. pytest ignores its count in code coverage.

`def extract(self, words: list, **kwargs): #pragma: no cover`